### PR TITLE
Adding datadog page to plugins page

### PR DESCRIPTION
**Pull request by padraigobrien**
I am adding a readme to the [marketing site](https://roadie.io/backstage/plugins/) so we can point to it in the [marketplace](https://backstage.io/plugins).

adding in a datadog.md and associated screenshots.

- [x] Ran through eslint
- [x] Ran tests, there are no tests
- [x] Ran yarn format against files that were changed

### CODE REVIEW

File: content/backstage/plugins/datadog.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+  - intro: Add a Datadog card to the overview tab of the EntityPage.tsx source file
+    language: typescript
+    code: |
+      // packages/app/src/components/catalog/EntityPage.tsx
+      import {
+            Router as DatadogRouter,
+            GraphWidget as DatadogGraphWidget,
+            isDatadogGraphAvailable as isDatadogWidgetAvailable,
+            } from '@roadiehq/backstage-plugin-datadog';
```
Comment (dtuite) [inline content/backstage/plugins/datadog.md:45]: The indentation on these lines is incorrect.

File: content/backstage/plugins/datadog.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+      const ServiceEntityPage = ({ entity }: { entity: Entity }) => (
+        &lt;EntityPageLayout>
+            &lt;EntityPageLayout.Content
+                path="/datadog/*"
+                title="datadog"
+                element={<DatadogRouter entity={entity} />}
+                />
```
Comment (dtuite) [inline content/backstage/plugins/datadog.md:53]: The indentation on these lines is incorrect.

File: content/backstage/plugins/datadog.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    code: |
+      // packages/app/src/plugins.ts
+        const OverviewContent = ({ entity }: { entity: Entity }) => (
+            &lt;Grid container spacing={3} alignItems="stretch">
+            ...
+            {isDatadogWidgetAvailable(entity) && (
+            &lt;Grid item>
+                &lt;DatadogGraphWidget entity={entity} />
+            &lt;/Grid>
+            )}
```
Comment (dtuite) [inline content/backstage/plugins/datadog.md:68]: The indentation on these lines is incorrect.

File: content/backstage/plugins/datadog.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+## Security
+
+A word of note regarding the security of the datadog dashboards and graphs.
+
+The instructions provided for sharing dashboards and graphs generate a URL.
+
+This URL is public to anyone who bears it.
+
+If obtained by another actor, it is usable by them.
+
```
Comment (dtuite) [inline content/backstage/plugins/datadog.md:153]: The phrasing is a little awkward in the security section. How about something like this?

> Sharing Datadog dashboards and graphs makes them public on the internet and accessible by anyone with the URL. Make sure you do not share these URLs with untrusted actors.

**Review by dtuite — CHANGES_REQUESTED** [review]
Indentation needs work

File: content/backstage/plugins/datadog.md (hunk 4 above)
Comment (padraigobrien) [reply content/backstage/plugins/datadog.md:151]: Updated as your text came across better.

File: content/backstage/plugins/datadog.md (hunk 1 above)
Comment (padraigobrien) [reply content/backstage/plugins/datadog.md]: Fixed and noted for future.

File: content/backstage/plugins/datadog.md (hunk 3 above)
Comment (padraigobrien) [reply content/backstage/plugins/datadog.md:70]: Fixed and noted for the future

File: content/backstage/plugins/datadog.md (hunk 2 above)
Comment (padraigobrien) [reply content/backstage/plugins/datadog.md]: Fixed and noted for the future

**Comment by dtuite** [comment]
@padraigobrien is it ready for another look?

**Comment by padraigobrien** [comment]
Yes. I compared it to the indentation on another plugin, Jenkins.md. I rephrased the security concerned as the suggestion read better.

**Comment by padraigobrien** [comment]
Yes. I compared it to the indentation on another plugin, Jenkins.md. I rephrased the security concerned as the suggestion read better.

**Review by dtuite — APPROVED** [review]

**Pull request merged**